### ISSUE TITLE
Clarify which decentralization is desirable

### ISSUE THREAD

**Issue by pdehaye** [CrowdNotifier/documents#13]
Previous "decentralized" contact tracing solutions have arguably been everything but the opposite. For instance:
* each nation-state would only approve one app, 
* two big hardware players essentially captured the protocol to {uni,bi}-laterally dumb it down, expand it or implement it at OS level.

There are pros and cons to this outcome in the Bluetooth case, but certainly the word "decentralized" has led to huge confusion and unproductive discussions. 

Based on past experience, please clarify what you mean here by "decentralized", and what you consider as a positive outcome with respect to this protocol. 

For instance, if this protocol was adopted in a week by Google and Apple, and in two weeks these two would announce of their own volition they have built (closed source) technology feeding {Google,Apple} Maps with aggregated anonymous heat maps, would this solution still be decentralized?

Given current governmental failures in some countries, one can even question whether health authorities should have the centralized power this protocol ascribes to them.

**Comment by wouterl** [comment]
What we would like to achieve with respect to this protocol is clarified in the explicit list of Security and Privacy requirements (starting on page 7).
